modules/react/src/components/basic-controls.js

```diff
@@ -28,20 +28,20 @@ import {
 } from '@hubble.gl/core';
 import EncoderDropdown from './encoder-dropdown';
 
-export default function BasicControls({adapter, busy, setBusy, encoderSettings}) {
+export default function BasicControls({adapter, busy, setBusy, encoderSettings, updateCamera}) {
```

Comment:
Can we make updateCamera an optional prop by giving it a default value (either undefined or a noop function)?